test/doc/core_1_refresher.cpp

```diff
@@ -325,7 +325,7 @@ namespace {
 //------------------------------------------------------------------------------
 
 //[code_core_1_refresher_8
-template <class AsyncWriteStream, class WriteHandler>
+template <class AsyncWriteStream, BOOST_BEAST_ASYNC_TPARAM2 WriteHandler>
```

Comment:
This shows up in the documentation, I don't think we want to show this to users. It should explicilty use `BOOST_ASIO_COMPLETION_TOKEN_FOR` or it should just say `class`.